hardhat/tasks/adminManagerRole.js

```diff
@@ -0,0 +1,33 @@
+const { task } = require("hardhat/config");
+const Logs = require("node-logs");
+const logger = new Logs().showInConsole(true);
+const { sendWithMultisig } = require("../../deployment/helpers/helpers");
+
+task("setAdminManager", "SetAdminManager to the contract that implement adminManagerRole")
+    .addOptionalParam("signer", "Signer name: 'signer' or 'deployer'", "deployer")
+    .addParam("adminManagerTarget", "Admin manager target: 'MultiSigWallet'")
+    .addParam("contractTarget", "Contract name target: e.g: 'VestingRegistry'")
```

Comment:
pls add support for address input - see multisig hh tasks for reference  
```suggestion
    .addParam("contract", "Contract name or address to set admin manager: e.g: 'VestingRegistry'")
```